plugins/modules/monitoring/pagerduty.py

```diff
@@ -228,8 +236,8 @@ def main():
             service=dict(required=False, type='list', aliases=["services"]),
             window_id=dict(required=False),
             requester_id=dict(required=False),
-            hours=dict(default='1', required=False),
-            minutes=dict(default='0', required=False),
+            hours=dict(default='1', required=False),   # @TODO change to int?
+            minutes=dict(default='0', required=False),   # @TODO change to int?
```

Comment:
Maybe it also allows fractional numbers. Who knows :)

Reply:
Anything but the string, man :-)